assets/js/add-task.js

```diff
@@ -37,7 +36,7 @@ function addTaskEvent(
 
     let taskObject = {
       content: inputValue.value,
```

Comment:
see https://github.com/rsa-wtag/Todo-App-with-JS/pull/2#discussion_r1161263653

```suggestion
      content: inputEl.value,
```